### DISCUSSION TITLE
Unlink my email from locked account

### DISCUSSION THREAD

**Discussion by y2003huang** [community/community#55757] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I failed to reset my gitHub account password because I lost my 2FA credential and recovered code. Could someone let me know how I can free my email from the locked account? 
Thanks

**Comment by airtower-luna** [comment]
See here: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/unlinking-your-email-address-from-a-locked-account

**Reply by y2003huang** [reply]
@airtower-luna Thanks for answer my question. I tried to follow the steps to unlink my email with the locked account. However, the steps don't work. The following steps were I did.
1. Opened link https://github.com/login as step 1
2. Entered username and password as step 2, I got the "Incorrect username or password" message, not "Unable to verify with your security key?", click Use a recovery code or request a reset as step 3

<img width="242" alt="pic2" src="https://github.com/community/community/assets/133034880/d6052935-e5b7-4b0c-b689-785798447a2e">

3. Then I clicked Forgot password to open Reset your password page and sent password reset email
4. Clicked Reset your password to open this page

<img width="220" alt="pic1" src="https://github.com/community/community/assets/133034880/2eff4fef-da47-4113-bb2e-78a226e9372c">

5. Because I lost my 2FA credential and recovery code, I clicked "two-factor recovery code" which opened the document "Configure 2FA recovery page".
I couldn't find where I can hit step 3.

**Reply by airtower-luna** [reply]
> Entered username and password as step 2, I got the "Incorrect username or password" message, not "Unable to verify with your security key?", click Use a recovery code or request a reset as step 3

The instructions in the documentation assume you can enter a correct password (step 2). :confused: 

I don't know what to do if you don't have that either, you might have to [ask GitHub support](https://support.github.com/request). Good luck!

**Reply by y2003huang** [reply]
@airtower-luna Do you have idea how I can reach out to GitHub support quickly? I submitted the help ticket for a while. However, I don't get response from them.

**Reply by airtower-luna** [reply]
Unfortunately not, I've read it can take a while. :confused:

**Comment by klabbers** [comment]
Same problem, I have one last shot, hoping my old laptop has the password in the browser or keychain.  I opened a ticket as well.  I have other e-mails, but it is frustrating.
